lib/order.rb

```diff
@@ -0,0 +1,90 @@
+require 'twilio-ruby'
+
+class Order
+
+    attr_reader :menu, :items
+
+    def initialize(menu)
+        @menu = menu
+        @items = []
+        @items_detail
+        @total = 0
+    end
+
+    def add(dish)
+        @items << dish
+        @items[-1]
+    end
+
+    def open
+        menu.list
+        select
+        if confirm
+            send_confirmation
+        end
+    end
+
+    def select
+        puts "Enter number of dish to add to order. Enter empty to complete."
+        while true do
+            item = STDIN.gets.chomp
+            break if item.empty?
+            items << item.to_i
+        end
+    end
+
+    def confirm
+        print
+        puts "Enter 'YES' to confirm order, leave blank to cancel order"
+        STDIN.gets.chomp.upcase == 'YES'
+    end
+
+    def print
+        summarise_order
+        puts "Your order summary is as follows:"
+        puts "---------------------------------"
+        @items_detail.each do |key, item|
```

Comment:
nice presentation of the order, dont think you need the puts though I may be wrong.